catalog/catalog-app/src/main/resources/features.xml

```diff
@@ -186,47 +164,56 @@
 
     <feature name="catalog-plugin-federationreplication" install="manual"
              version="${project.version}" description="">
+            <feature prerequisite="true">catalog-app</feature>
         <bundle>mvn:ddf.catalog.plugin/plugin-federation-replication/${project.version}</bundle>
     </feature>
 
-    <feature name="catalog-plugin-jpeg2000" install="manual" version="${project.version}"
+    <feature name="catalog-plugin-jpeg2000" install="auto" version="${project.version}"
```

Comment:
I see that this had been installed as part of `catalog-app` before, but should it really be started by default?